utility/src/main/java/org/oppia/util/firebase/CrashLoggerImplementation.kt

```diff
@@ -0,0 +1,16 @@
+package org.oppia.util.firebase
+
+import com.google.firebase.crashlytics.FirebaseCrashlytics
+import javax.inject.Singleton
+
+/** Logger for providing custom crash reporting to Firebase Crashlytics. */
+@Singleton
+open class CrashLoggerImplementation(
```

Comment:
This class isn't actually being extended, so suggest removing the ``open`` property to prefer the default state of immutability.